From 170606b94c137684b0b875512b86e872a6efbe2b Mon Sep 17 00:00:00 2001
Message-Id: <170606b94c137684b0b875512b86e872a6efbe2b.1376492227.git.minovotn@redhat.com>
In-Reply-To: <276ddced7c9181cce17d0ff9eb080f99dcfe0ac3.1376492227.git.minovotn@redhat.com>
References: <276ddced7c9181cce17d0ff9eb080f99dcfe0ac3.1376492227.git.minovotn@redhat.com>
From: Asias He <asias@redhat.com>
Date: Wed, 14 Aug 2013 10:24:10 +0200
Subject: [PATCH 09/22] qemu-tool: Add dummy qemu_mutex_lock_iothread and
 qemu_mutex_unlock_iothread

RH-Author: Asias He <asias@redhat.com>
Message-id: <1376475863-27929-5-git-send-email-asias@redhat.com>
Patchwork-id: 53391
O-Subject: [RHEL6.5 qemu-kvm PATCH v4 04/17] qemu-tool: Add dummy qemu_mutex_lock_iothread and qemu_mutex_unlock_iothread
Bugzilla: 848070
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>

This is needed by gluster driver with qemu-img, qemu-io, etc.

This is a RHEL6 downstream only patch.

Signed-off-by: Asias He <asias@redhat.com>
---
 qemu-tool.c | 8 ++++++++
 1 file changed, 8 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-tool.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/qemu-tool.c b/qemu-tool.c
index e05a0c2..a09f655 100644
--- a/qemu-tool.c
+++ b/qemu-tool.c
@@ -117,3 +117,11 @@ bool runstate_check(RunState state)
 {
     return state == RUN_STATE_RUNNING;
 }
+
+void qemu_mutex_lock_iothread(void)
+{
+}
+
+void qemu_mutex_unlock_iothread(void)
+{
+}
-- 
1.7.11.7

